vizro-core/docs/pages/user-guides/tabs.md

```diff
@@ -138,4 +138,133 @@ To add [`Tabs`][vizro.models.Tabs] to your page, do the following:
 
         [![Tabs]][tabs]
 
+## Add a tooltip
+
+The `description` argument enables you to add helpful context to your tabs by displaying an info icon next to its title. Hovering over the icon shows a tooltip with your chosen text.
```

Comment:
Maybe nice to have a screengrab here. That way, people don't have to try out the whole dashboard below just to find out if it is what they think it is. They can get the sample look and feel from the static image.

Reply:
I would leave this as you have it is since it matches how we document it elsewhere and otherwise we would need to add the picture everywhere, which is more effort than it's worth 😅 We have the picture in a tab in the below example and I think that's enough.